tests/kafkatest/tests/streams/streams_broker_down_resilience_test.py

```diff
@@ -180,24 +226,40 @@ def test_streams_should_failover_while_brokers_down(self):
         processor_2.start()
 
         processor_3 = StreamsBrokerDownResilienceService(self.test_context, self.kafka, configs)
-        processor_3.start()
 
         # need to wait for rebalance once
-        self.wait_for_verification(processor_3, "State transition from REBALANCING to RUNNING", processor_3.LOG_FILE)
-
-        # assert streams can process when starting with broker up
-        self.assert_produce_consume(self.inputTopic,
-                                    self.outputTopic,
-                                    self.client_id,
-                                    "waiting for rebalance to complete",
-                                    num_messages=9,
-                                    timeout_sec=120)
-
-        message = "processed3messages"
-
-        self.wait_for_verification(processor, message, processor.STDOUT_FILE)
-        self.wait_for_verification(processor_2, message, processor_2.STDOUT_FILE)
-        self.wait_for_verification(processor_3, message, processor_3.STDOUT_FILE)
+        rebalance = "State transition from REBALANCING to RUNNING"
```

Comment:
Why only monitor on processor_3 not the previous two?

Reply:
Since processor 3 is the last streams client to join the group, once it has transitioned from `REBALANCE to RUNNING` the test is ready to proceed.